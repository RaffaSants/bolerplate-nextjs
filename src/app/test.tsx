import { render, screen } from '@testing-library/react'

import Home from './page'

describe('<Home />', () => {
  it('should render the heading', () => {
    const { container } = render(<Home />)

    expect(
      screen.getByRole('heading', { name: /this is a bolerplate!/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
