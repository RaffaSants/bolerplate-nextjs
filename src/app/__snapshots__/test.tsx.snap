// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the heading 1`] = `
<h1>
  This is a Bolerplate!
</h1>
`
